packages/pieces/community/acumbamail/src/lib/actions/really-delete-subscriber.ts

```diff
@@ -0,0 +1,38 @@
+import { acumbamailAuth } from '../../';
+import { Property, createAction } from '@activepieces/pieces-framework';
+import { acumbamailCommon } from '../common';
+import {
+  HttpRequest,
+  HttpMethod,
+  httpClient,
+} from '@activepieces/pieces-common';
+
+export const reallyDeleteSubscriberAction = createAction({
+  auth: acumbamailAuth,
+  name: 'acumbamail_really_delete_subscriber',
+  displayName: 'Remove Subscriber',
+  description:
+    'Removes a subscriber from a list',
```

Comment:
Do you delete the subscriber from list or do a hard delete for the subscriber? what is the different :D

Reply:
apparently there is a difference lol. These are two different actions. One removes the email from a list, and the other one, keeps the email in the list but... marked as unsubscribed? haha